docs/guides/preparing/local-server-setup.mdx

```diff
@@ -0,0 +1,302 @@
+---
+id: local-server-setup
+title: Setting Up a Server
+---
+
+import ImageSwitcher from "@site/src/components/ImageSwitcher.js";
+
+Setting up a local Minecraft server loaded with PGM is one of the best ways to test your map without disruption and allows for greater experimentation. 
+Additionally, being able to test maps before submitting to a Minecraft network of your choice means that you will be able to catch mistakes early and speed up the map submission process.
+
+:::note
+This guide is divided into three most commonly used operating systems. Please keep in mind that some steps may differ or may not apply to your system.
+If you have any questions or need assistance, please visit our Discord server.
+:::
+
+## Windows
+
+This guide uses Windows 11, however, the same process will work on Windows 10 and older.
+
+### Installing Java
+
+We recommend using either Java 8 or Java 11. For this guide, we will be using [Amazon Corretto 11](https://aws.amazon.com/corretto/). 
```

Comment:
The server works with Java 11?

Reply:
Yes, Pablo has recommended it a few times in the PGM Discord.